src/appBootstrapper.jsx

```diff
@@ -96,6 +96,7 @@ window.FRONTEND_COMMIT = COMMIT;
 // portal header
 window.postLoadForMskCIS = function() {
     AppConfig.hide_login = true;
+    window.isMSKCIS = true;
```

Comment:
kinda hacky but i don't think we want to formalize something that is already hacked in for MSK only